### ISSUE TITLE
Link for patched version

### ISSUE THREAD

**Issue by KarlMaximus** [wheatevo/wiiu-vc-extractor#49]
Hello! Sometimes extracting gba vc roms from alldata.psb.m may fail due to spaces between internal names. It was fixed back then according to this thread (https://github.com/wheatevo/wiiu-vc-extractor/pull/13). But the problem is, while the fix was done by editing "GbaVcExtractor.cs", it made difficult for end users as some knowledge of programming (C#) is required to apply the fix to the Release Version. Hence it would be nice if you could give instructions on how to apply the fix on the Release Version, or better, link an already-fixed version for us dummies.

**Comment by wheatevo** [comment]
This was actually resolved after #13 was merged in the 0.4.2 release. The current [0.9.0 release](https://github.com/wheatevo/wiiu-vc-extractor/releases/tag/0.9.0) has a similar modification that allows all title characters:
```cs
if ((gbaHeader[i] < ASCII_SPACE || gbaHeader[i] > ASCII_TILDE) && gbaHeader[i] != 0x00)
```
from https://github.com/wheatevo/wiiu-vc-extractor/blob/0.9.0/WiiuVcExtractor/RomExtractors/GbaVcExtractor.cs#L196

Are you seeing issues with GBA extraction in the 0.9.0 release? If so, could you please post your verbose output into this issue?

**Comment by KarlMaximus** [comment]
Appreciate for the prompt reply. The following is what I am getting after using the 0.9.0 release to extract the "Onimusha Tactics" gba rom from alldata.psb.m and alldata.bin.

**PS C:\Users\ASUS\Desktop\wiiu-vc-extractor-0.9.0> .\wiiuvcextractor [-v] alldata.psb.m
============================================================================
Starting extraction of rom from alldata.psb.m...
============================================================================
PSB file detected!
Decompressing PSB file...
Decompression to alldata.psb.m.extract completed
Checking for PSB data file C:\Users\ASUS\Desktop\wiiu-vc-extractor-0.9.0\alldata.bin...
Found rom subfile at aa6oe0.d44.m
    Offset: 33376256
    Length: 2787153
Decompressing rom...
Decompressing PSB file...
Decompression to C:\Users\ASUS\Desktop\wiiu-vc-extractor-0.9.0\aa6oe0.d44.m.extract completed
Checking if this is a GBA VC title...
Checking C:\Users\ASUS\Desktop\wiiu-vc-extractor-0.9.0\aa6oe0.d44.m.extract...
Could not find decompressed rom at C:\Users\ASUS\Desktop\wiiu-vc-extractor-0.9.0\aa6oe0.d44.m.extract
============================================================================
FAILURE: Could not successfully identify the rom type for alldata.psb.m
============================================================================
PS C:\Users\ASUS\Desktop\wiiu-vc-extractor-0.9.0>**

For comparison, I could extract the same rom without problem using "inject_gba" so I am sure the source files aren't corrupted. In addition, I also tried with "Final Fantasy Tactics Advance" which gives the same failure message.

**PS C:\Users\ASUS\Desktop\wiiu-vc-extractor-0.9.0> .\wiiuvcextractor [-v] alldata.psb.m
============================================================================
Starting extraction of rom from alldata.psb.m...
============================================================================
PSB file detected!
Decompressing PSB file...
Decompression to alldata.psb.m.extract completed
Checking for PSB data file C:\Users\ASUS\Desktop\wiiu-vc-extractor-0.9.0\alldata.bin...
Found rom subfile at aafxe0.c21.m
    Offset: 35997696
    Length: 5732577
Decompressing rom...
Decompressing PSB file...
Decompression to C:\Users\ASUS\Desktop\wiiu-vc-extractor-0.9.0\aafxe0.c21.m.extract completed
Checking if this is a GBA VC title...
Checking C:\Users\ASUS\Desktop\wiiu-vc-extractor-0.9.0\aafxe0.c21.m.extract...
Could not find decompressed rom at C:\Users\ASUS\Desktop\wiiu-vc-extractor-0.9.0\aafxe0.c21.m.extract
============================================================================
FAILURE: Could not successfully identify the rom type for alldata.psb.m
============================================================================
PS C:\Users\ASUS\Desktop\wiiu-vc-extractor-0.9.0>**

**Comment by wheatevo** [comment]
Could you please re-run the tests with the `-v` flag listed without the brackets?

For example:
```sh
.\wiiuvcextractor -v alldata.psb.m
```

We should be able to see what's going on once we have output from there. This might be similar to some of the issues that were seen in #33.

**Comment by KarlMaximus** [comment]
I checked and found some interesting things.

When I ran with either ".\wiiuvcextractor [-v] alldata.psb.m" or ".\wiiuvcextractor alldata.psb.m" command, both roms (Onimusha Tactics and Final Fantasy Advance Tactics) cannot be extracted.

When I ran with your suggested ".\wiiuvcextractor -v alldata.psb.m" command, I could extract "Final Fantasy Tactics Advance" rom after a long string of texts. But when I tried to extract "Onimusha Tactics", the process failed with the same error as those above but with extended wall of texts this time. This is what I was getting from it:

PS C:\Users\ASUS\Desktop\wiiu-vc-extractor-0.9.0> .\wiiuvcextractor -v alldata.psb.m
Verbose output mode is set
Source extract file is C:\Users\ASUS\Desktop\wiiu-vc-extractor-0.9.0\alldata.psb.m
============================================================================
Starting extraction of rom from alldata.psb.m...
============================================================================
PSB file detected!
Attempting to read PSB file from alldata.psb.m...
Decompressing PSB file...
MDF Header content:
MdfHeader:
signature: 6D-64-66-00
length: 20286

Generating XOR key for MDF decryption...
Reading bytes from alldata.psb.m...
Decrypting MDF data...
Decryption complete
Decompressing MDF data...
Decompression to alldata.psb.m.extract completed
PSB Header information:
PsbHeader:
signature: 50-53-42-00
type: 2
unknown: 0
namesOffset: 40
stringsOffset: 20268
stringsDataOffset: 20272
chunkOffsetsOffset: 20280
chunkLengthsOffset: 20283
chunkDataOffset: 20286
entriesOffset: 15672

Checking for PSB data file C:\Users\ASUS\Desktop\wiiu-vc-extractor-0.9.0\alldata.bin...
Checking if 0781/config/title_menu_motion.psb.m contains the rom subfile path...
Checking if 0781/config/title_menu_string.psb.m contains the rom subfile path...
Checking if 0781/config/title_mode_setting_pad.psb.m contains the rom subfile path...
Checking if 0781/config/title_prof.psb.m contains the rom subfile path...
Checking if 0781/config/title_prof_specdepend.psb.m contains the rom subfile path...
Checking if 0781/config/title_savedata_string.psb.m contains the rom subfile path...
Checking if 0781/config/title_session.psb.m contains the rom subfile path...
Checking if 0781/config/title_setting_game.psb.m contains the rom subfile path...
Checking if 0781/jpeg/page01.jpg.m contains the rom subfile path...
Checking if 0781/jpeg/page02.jpg.m contains the rom subfile path...
Checking if 0781/jpeg/page03.jpg.m contains the rom subfile path...
Checking if 0781/jpeg/page04.jpg.m contains the rom subfile path...
Checking if 0781/jpeg/page05.jpg.m contains the rom subfile path...
Checking if 0781/jpeg/page06.jpg.m contains the rom subfile path...
Checking if 0781/jpeg/page07.jpg.m contains the rom subfile path...
Checking if 0781/jpeg/page08.jpg.m contains the rom subfile path...
Checking if 0781/jpeg/page09.jpg.m contains the rom subfile path...
Checking if 0781/jpeg/page10.jpg.m contains the rom subfile path...
Checking if 0781/jpeg/page11.jpg.m contains the rom subfile path...
Checking if 0781/jpeg/page12.jpg.m contains the rom subfile path...
Checking if 0781/jpeg/page13.jpg.m contains the rom subfile path...
Checking if 0781/jpeg/page14.jpg.m contains the rom subfile path...
Checking if 0781/jpeg/page15.jpg.m contains the rom subfile path...
Checking if 0781/jpeg/page16.jpg.m contains the rom subfile path...
Checking if 0781/jpeg/page17.jpg.m contains the rom subfile path...
Checking if 0781/jpeg/page18.jpg.m contains the rom subfile path...
Checking if 0781/jpeg/page19.jpg.m contains the rom subfile path...
Checking if 0781/jpeg/page20.jpg.m contains the rom subfile path...
Checking if 0781/motion/bg01.psb.m contains the rom subfile path...
Checking if 0781/script/title_session.nut.m contains the rom subfile path...
Checking if 0781/script/title_standalone.nut.m contains the rom subfile path...
Checking if system/config/backup_statedata_confirm.psb.m contains the rom subfile path...
Checking if system/config/font_icon_code.psb.m contains the rom subfile path...
Checking if system/config/game_content.psb.m contains the rom subfile path...
Checking if system/config/genre_code.psb.m contains the rom subfile path...
Checking if system/config/guide_body.psb.m contains the rom subfile path...
Checking if system/config/guide_parts.psb.m contains the rom subfile path...
Checking if system/config/kinsoku.psb.m contains the rom subfile path...
Checking if system/config/manual_viewer_vessel.psb.m contains the rom subfile path...
Checking if system/config/menu_cursor.psb.m contains the rom subfile path...
Checking if system/config/menu_info.psb.m contains the rom subfile path...
Checking if system/config/menu_item.psb.m contains the rom subfile path...
Checking if system/config/menu_motion_text.psb.m contains the rom subfile path...
Checking if system/config/menu_rankingattack.psb.m contains the rom subfile path...
Checking if system/config/menu_str_replace.psb.m contains the rom subfile path...
Checking if system/config/mode_change_title.psb.m contains the rom subfile path...
Checking if system/config/mode_command.psb.m contains the rom subfile path...
Checking if system/config/mode_game_play.psb.m contains the rom subfile path...
Checking if system/config/mode_help_and_options.psb.m contains the rom subfile path...
Checking if system/config/mode_leaderboard.psb.m contains the rom subfile path...
Checking if system/config/mode_main.psb.m contains the rom subfile path...
Checking if system/config/mode_manual.psb.m contains the rom subfile path...
Checking if system/config/mode_misc.psb.m contains the rom subfile path...
Checking if system/config/mode_option.psb.m contains the rom subfile path...
Checking if system/config/mode_pause.psb.m contains the rom subfile path...
Checking if system/config/mode_ranking.psb.m contains the rom subfile path...
Checking if system/config/mode_replay.psb.m contains the rom subfile path...
Checking if system/config/mode_setting_pad_default.psb.m contains the rom subfile path...
Checking if system/config/mode_soundmode.psb.m contains the rom subfile path...
Checking if system/config/notice_europe.psb.m contains the rom subfile path...
Checking if system/config/notice_japan.psb.m contains the rom subfile path...
Checking if system/config/notice_usa.psb.m contains the rom subfile path...
Checking if system/config/pause_help_and_options.psb.m contains the rom subfile path...
Checking if system/config/pause_main.psb.m contains the rom subfile path...
Checking if system/config/pause_match.psb.m contains the rom subfile path...
Checking if system/config/pause_option.psb.m contains the rom subfile path...
Checking if system/config/pause_rankingattack.psb.m contains the rom subfile path...
Checking if system/config/play_rankingattack.psb.m contains the rom subfile path...
Checking if system/config/play_standalone_sub.psb.m contains the rom subfile path...
Checking if system/config/rating.psb.m contains the rom subfile path...
Checking if system/config/rating_country.psb.m contains the rom subfile path...
Checking if system/config/replay_play.psb.m contains the rom subfile path...
Checking if system/config/savedata_string.psb.m contains the rom subfile path...
Checking if system/config/session.psb.m contains the rom subfile path...
Checking if system/config/session_rule_info.psb.m contains the rom subfile path...
Checking if system/config/session_ui_search_list.psb.m contains the rom subfile path...
Checking if system/config/session_ui_setting.psb.m contains the rom subfile path...
Checking if system/config/setting_etc.psb.m contains the rom subfile path...
Checking if system/config/setting_network.psb.m contains the rom subfile path...
Checking if system/config/setting_others.psb.m contains the rom subfile path...
Checking if system/config/setting_pad_id.psb.m contains the rom subfile path...
Checking if system/config/setting_savedata.psb.m contains the rom subfile path...
Checking if system/config/setting_screen.psb.m contains the rom subfile path...
Checking if system/config/setting_sound.psb.m contains the rom subfile path...
Checking if system/config/struct_picture_bitmap.psb.m contains the rom subfile path...
Checking if system/config/struct_replaydata.psb.m contains the rom subfile path...
Checking if system/config/struct_rule_playside.psb.m contains the rom subfile path...
Checking if system/config/struct_rule_setting.psb.m contains the rom subfile path...
Checking if system/config/struct_savedata_dummy.psb.m contains the rom subfile path...
Checking if system/config/struct_setting_game_image.psb.m contains the rom subfile path...
Checking if system/config/struct_sound_play_list.psb.m contains the rom subfile path...
Checking if system/config/struct_sram_image.psb.m contains the rom subfile path...
Checking if system/config/struct_statedata.psb.m contains the rom subfile path...
Checking if system/config/struct_statedata_extend.psb.m contains the rom subfile path...
Checking if system/config/struct_systemdata.psb.m contains the rom subfile path...
Checking if system/config/struct_systemdata_title.psb.m contains the rom subfile path...
Checking if system/config/struct_systemdata_titles.psb.m contains the rom subfile path...
Checking if system/config/system_prof.psb.m contains the rom subfile path...
Checking if system/font/multi32_0.psb.m contains the rom subfile path...
Checking if system/font/multi_warning.psb.m contains the rom subfile path...
Checking if system/motion/bg00.psb.m contains the rom subfile path...
Checking if system/motion/dialog_base.psb.m contains the rom subfile path...
Checking if system/motion/dialog_europe_dut.psb.m contains the rom subfile path...
Checking if system/motion/dialog_europe_eng.psb.m contains the rom subfile path...
Checking if system/motion/dialog_europe_fra.psb.m contains the rom subfile path...
Checking if system/motion/dialog_europe_ger.psb.m contains the rom subfile path...
Checking if system/motion/dialog_europe_ita.psb.m contains the rom subfile path...
Checking if system/motion/dialog_europe_por.psb.m contains the rom subfile path...
Checking if system/motion/dialog_europe_rus.psb.m contains the rom subfile path...
Checking if system/motion/dialog_europe_spa.psb.m contains the rom subfile path...
Checking if system/motion/dialog_japan_jpn.psb.m contains the rom subfile path...
Checking if system/motion/dialog_usa_eng.psb.m contains the rom subfile path...
Checking if system/motion/dialog_usa_fra.psb.m contains the rom subfile path...
Checking if system/motion/dialog_usa_por.psb.m contains the rom subfile path...
Checking if system/motion/dialog_usa_spa.psb.m contains the rom subfile path...
Checking if system/motion/keyconfig_base.psb.m contains the rom subfile path...
Checking if system/motion/keyconfig_europe_dut.psb.m contains the rom subfile path...
Checking if system/motion/keyconfig_europe_eng.psb.m contains the rom subfile path...
Checking if system/motion/keyconfig_europe_fra.psb.m contains the rom subfile path...
Checking if system/motion/keyconfig_europe_ger.psb.m contains the rom subfile path...
Checking if system/motion/keyconfig_europe_ita.psb.m contains the rom subfile path...
Checking if system/motion/keyconfig_europe_por.psb.m contains the rom subfile path...
Checking if system/motion/keyconfig_europe_rus.psb.m contains the rom subfile path...
Checking if system/motion/keyconfig_europe_spa.psb.m contains the rom subfile path...
Checking if system/motion/keyconfig_japan_jpn.psb.m contains the rom subfile path...
Checking if system/motion/keyconfig_usa_eng.psb.m contains the rom subfile path...
Checking if system/motion/keyconfig_usa_fra.psb.m contains the rom subfile path...
Checking if system/motion/keyconfig_usa_por.psb.m contains the rom subfile path...
Checking if system/motion/keyconfig_usa_spa.psb.m contains the rom subfile path...
Checking if system/motion/logo_m2.psb.m contains the rom subfile path...
Checking if system/motion/main_menu_base.psb.m contains the rom subfile path...
Checking if system/motion/main_menu_europe_dut.psb.m contains the rom subfile path...
Checking if system/motion/main_menu_europe_eng.psb.m contains the rom subfile path...
Checking if system/motion/main_menu_europe_fra.psb.m contains the rom subfile path...
Checking if system/motion/main_menu_europe_ger.psb.m contains the rom subfile path...
Checking if system/motion/main_menu_europe_ita.psb.m contains the rom subfile path...
Checking if system/motion/main_menu_europe_por.psb.m contains the rom subfile path...
Checking if system/motion/main_menu_europe_rus.psb.m contains the rom subfile path...
Checking if system/motion/main_menu_europe_spa.psb.m contains the rom subfile path...
Checking if system/motion/main_menu_japan_jpn.psb.m contains the rom subfile path...
Checking if system/motion/main_menu_usa_eng.psb.m contains the rom subfile path...
Checking if system/motion/main_menu_usa_fra.psb.m contains the rom subfile path...
Checking if system/motion/main_menu_usa_por.psb.m contains the rom subfile path...
Checking if system/motion/main_menu_usa_spa.psb.m contains the rom subfile path...
Checking if system/motion/manual_viewer.psb.m contains the rom subfile path...
Checking if system/motion/nowloading.psb.m contains the rom subfile path...
Checking if system/motion/wipe.psb.m contains the rom subfile path...
Checking if system/roms/AA6OE0.D44.m contains the rom subfile path...
Found rom subfile at aa6oe0.d44.m
    Offset: 33376256
    Length: 2787153
Checking if system/script/autosave_wait_dialog.nut.m contains the rom subfile path...
Checking if system/script/backup.nut.m contains the rom subfile path...
Checking if system/script/backup_playlist.nut.m contains the rom subfile path...
Checking if system/script/backup_replaydata.nut.m contains the rom subfile path...
Checking if system/script/backup_specdepend.nut.m contains the rom subfile path...
Checking if system/script/backup_statedata.nut.m contains the rom subfile path...
Checking if system/script/backup_statedata_confirm.nut.m contains the rom subfile path...
Checking if system/script/const.nut.m contains the rom subfile path...
Checking if system/script/dialog.nut.m contains the rom subfile path...
Checking if system/script/emulator.nut.m contains the rom subfile path...
Checking if system/script/emulator_attr.nut.m contains the rom subfile path...
Checking if system/script/emulator_sram.nut.m contains the rom subfile path...
Checking if system/script/game_content.nut.m contains the rom subfile path...
Checking if system/script/init.nut.m contains the rom subfile path...
Checking if system/script/init_specdepend.nut.m contains the rom subfile path...
Checking if system/script/init_sub.nut.m contains the rom subfile path...
Checking if system/script/itemindicator.nut.m contains the rom subfile path...
Checking if system/script/manual_viewer_vessel.nut.m contains the rom subfile path...
Checking if system/script/manual_viewer_vessel_tuning.nut.m contains the rom subfile path...
Checking if system/script/medal_system.nut.m contains the rom subfile path...
Checking if system/script/menu_bg.nut.m contains the rom subfile path...
Checking if system/script/menu_frame_base.nut.m contains the rom subfile path...
Checking if system/script/menu_guide.nut.m contains the rom subfile path...
Checking if system/script/menu_info.nut.m contains the rom subfile path...
Checking if system/script/menu_motion.nut.m contains the rom subfile path...
Checking if system/script/menu_motion_text.nut.m contains the rom subfile path...
Checking if system/script/menu_motion_vessel.nut.m contains the rom subfile path...
Checking if system/script/menu_select_list.nut.m contains the rom subfile path...
Checking if system/script/menu_setting.nut.m contains the rom subfile path...
Checking if system/script/menu_setting_subitem.nut.m contains the rom subfile path...
Checking if system/script/menu_sound.nut.m contains the rom subfile path...
Checking if system/script/menu_subitem_window.nut.m contains the rom subfile path...
Checking if system/script/menu_title_ticker.nut.m contains the rom subfile path...
Checking if system/script/mode_base.nut.m contains the rom subfile path...
Checking if system/script/mode_command.nut.m contains the rom subfile path...
Checking if system/script/mode_game_play.nut.m contains the rom subfile path...
Checking if system/script/mode_help_and_options.nut.m contains the rom subfile path...
Checking if system/script/mode_leaderboard.nut.m contains the rom subfile path...
Checking if system/script/mode_logo.nut.m contains the rom subfile path...
Checking if system/script/mode_main.nut.m contains the rom subfile path...
Checking if system/script/mode_manual.nut.m contains the rom subfile path...
Checking if system/script/mode_misc.nut.m contains the rom subfile path...
Checking if system/script/mode_network.nut.m contains the rom subfile path...
Checking if system/script/mode_network_sub.nut.m contains the rom subfile path...
Checking if system/script/mode_normal.nut.m contains the rom subfile path...
Checking if system/script/mode_opening.nut.m contains the rom subfile path...
Checking if system/script/mode_option.nut.m contains the rom subfile path...
Checking if system/script/mode_pause.nut.m contains the rom subfile path...
Checking if system/script/mode_ranking.nut.m contains the rom subfile path...
Checking if system/script/mode_replay.nut.m contains the rom subfile path...
Checking if system/script/mode_setting_pad.nut.m contains the rom subfile path...
Checking if system/script/mode_staff.nut.m contains the rom subfile path...
Checking if system/script/mode_title.nut.m contains the rom subfile path...
Checking if system/script/motion_process.nut.m contains the rom subfile path...
Checking if system/script/notice_dialog.nut.m contains the rom subfile path...
Checking if system/script/pause_help_and_options.nut.m contains the rom subfile path...
Checking if system/script/pause_main.nut.m contains the rom subfile path...
Checking if system/script/pause_match.nut.m contains the rom subfile path...
Checking if system/script/pause_option.nut.m contains the rom subfile path...
Checking if system/script/pause_rankingattack.nut.m contains the rom subfile path...
Checking if system/script/play_match.nut.m contains the rom subfile path...
Checking if system/script/play_rankingattack.nut.m contains the rom subfile path...
Checking if system/script/play_soundmode.nut.m contains the rom subfile path...
Checking if system/script/play_standalone.nut.m contains the rom subfile path...
Checking if system/script/play_standalone_sub.nut.m contains the rom subfile path...
Checking if system/script/popup_dialog.nut.m contains the rom subfile path...
Checking if system/script/preload.nut.m contains the rom subfile path...
Checking if system/script/ranking_manager.nut.m contains the rom subfile path...
Checking if system/script/ranking_manager_specdepend.nut.m contains the rom subfile path...
Checking if system/script/replay_play.nut.m contains the rom subfile path...
Checking if system/script/session_base.nut.m contains the rom subfile path...
Checking if system/script/session_common.nut.m contains the rom subfile path...
Checking if system/script/session_rule.nut.m contains the rom subfile path...
Checking if system/script/session_rule_info.nut.m contains the rom subfile path...
Checking if system/script/session_specdepend.nut.m contains the rom subfile path...
Checking if system/script/session_ui_member.nut.m contains the rom subfile path...
Checking if system/script/session_ui_search_list.nut.m contains the rom subfile path...
Checking if system/script/session_ui_setting.nut.m contains the rom subfile path...
Checking if system/script/setting_base.nut.m contains the rom subfile path...
Checking if system/script/setting_etc.nut.m contains the rom subfile path...
Checking if system/script/setting_game.nut.m contains the rom subfile path...
Checking if system/script/setting_network.nut.m contains the rom subfile path...
Checking if system/script/setting_others.nut.m contains the rom subfile path...
Checking if system/script/setting_pad_rapid.nut.m contains the rom subfile path...
Checking if system/script/setting_savedata.nut.m contains the rom subfile path...
Checking if system/script/setting_screen.nut.m contains the rom subfile path...
Checking if system/script/setting_screen_sub.nut.m contains the rom subfile path...
Checking if system/script/setting_screen_sys.nut.m contains the rom subfile path...
Checking if system/script/setting_sound.nut.m contains the rom subfile path...
Checking if system/script/systemdata.nut.m contains the rom subfile path...
Checking if system/script/systemdata_backup_flags.nut.m contains the rom subfile path...
Checking if system/script/systemdata_purchase_record.nut.m contains the rom subfile path...
Checking if system/script/systemdata_saveload.nut.m contains the rom subfile path...
Checking if system/script/systemdata_setting_etc.nut.m contains the rom subfile path...
Checking if system/script/systemdata_setting_game.nut.m contains the rom subfile path...
Checking if system/script/systemdata_setting_network.nut.m contains the rom subfile path...
Checking if system/script/systemdata_setting_pad.nut.m contains the rom subfile path...
Checking if system/script/systemdata_setting_savedata.nut.m contains the rom subfile path...
Checking if system/script/systemdata_setting_screen.nut.m contains the rom subfile path...
Checking if system/script/systemdata_setting_sound.nut.m contains the rom subfile path...
Checking if system/script/systemdata_sram_data.nut.m contains the rom subfile path...
Checking if system/script/systemdata_window_info.nut.m contains the rom subfile path...
Checking if system/script/systemdata_work_medal.nut.m contains the rom subfile path...
Checking if system/script/systemdata_work_trial.nut.m contains the rom subfile path...
Checking if system/script/utils.nut.m contains the rom subfile path...
Checking if system/script/wipe.nut.m contains the rom subfile path...
Checking if system/sound/menu_sound.psb contains the rom subfile path...
Checking if system/sound/se.psb contains the rom subfile path...
Decompressing rom...
Decompressing PSB file...
Decompression to C:\Users\ASUS\Desktop\wiiu-vc-extractor-0.9.0\aa6oe0.d44.m.extract completed
Checking if this is a GBA VC title...
Checking C:\Users\ASUS\Desktop\wiiu-vc-extractor-0.9.0\aa6oe0.d44.m.extract...
Could not find decompressed rom at C:\Users\ASUS\Desktop\wiiu-vc-extractor-0.9.0\aa6oe0.d44.m.extract
============================================================================
FAILURE: Could not successfully identify the rom type for alldata.psb.m
============================================================================
PS C:\Users\ASUS\Desktop\wiiu-vc-extractor-0.9.0>

**Comment by Lucaboom16** [comment]
Hi, I have been having problems with the new version of Wiiuvcextractor. This is my first time using this but first, do I use command prompt? If I am not supposed to, the wiiuvcextractor.exe is not opening up. So please tell me what I should do. and maybe tell me the code for a SNES .rpx file, please. thanks.

**Comment by wheatevo** [comment]
Yes, you should be running it from a command prompt. Could you please file a new issue with the output you are seeing when you run the executable?

**Comment by Lucaboom16** [comment]
yes, it says exactly: ' wiiuvcextractor' is not recognized as an internal or external command, operable program, or batch file. The command I put for my SNES super Mario world game was c:\users\owner>wiiuvcextractor WUP-JAAE.rpx

**Comment by wheatevo** [comment]
In this case assuming wiiuvcextractor is in your current directory, you would just need to prepend `.\` to it:
```
.\wiiuvcextractor WUP-JAAE.rpx
```

**Comment by Lucaboom16** [comment]
For some reason, it says the same thing. It doesn't work. I am doing what you are saying. I opened up command prompt, even copy and pasted the command you sent, and it still says  ' .\wiiuvcextractor' is not recognized as an internal or external command, operable program, or batch file. The command looks like: c:\users\owner>.\wiiuvcextractor WUP-JAAE.rpx did I do something wrong?

**Comment by Lucaboom16** [comment]
Ya I kept trying different things and it isn't working.

**Comment by wheatevo** [comment]
Have you confirmed that wiiuvcextractor has been extracted to the c:\users\owner directory?

**Comment by Lucaboom16** [comment]
Ok I am dumb I just needed to take it out of the folder thank you

**Comment by Lucaboom16** [comment]
also can I turn the .sfc into a .cia file?

**Comment by wheatevo** [comment]
Closing this issue since the original problem was resolved.
